instrumented/nightly-tests/src/androidTest/kotlin/com/datadog/android/nightly/rum/RumConfigE2ETests.kt

```diff
@@ -375,6 +375,28 @@ class RumConfigE2ETests {
         sendRandomActionEvent(testMethodName)
     }
 
+    /**
+     * apiMethodSignature: com.datadog.android.core.configuration.Configuration$Builder#fun disableInteractionTracking(): Builder
+     */
+    @Test
+    fun rum_config_disable_rum_action_tracker() {
+        val testMethodName = "rum_config_disable_rum_action_tracker"
+        measureSdkInitialize {
+            initializeSdk(
+                InstrumentationRegistry.getInstrumentation().targetContext,
+                TrackingConsent.GRANTED,
+                defaultConfigurationBuilder(
+                    logsEnabled = true,
+                    tracesEnabled = true,
+                    rumEnabled = true,
+                    crashReportsEnabled = true
+                ).disableInteractionTracking().build()
+            )
+        }
+
+        sendRandomActionEvent(testMethodName)
```

Comment:
It seems we don't have a monitor for this test yet, right? Also I'm curious if we can test `disableInteractionTracking ` at all - `sendRandomActionEvent` has no connection to this method it seems, because it will send a manual action, which is not guarded by the `disableInteractionTracking`.

Reply:
I was going to add the monitor after this is in, since creating it early would just cause some trigger noise.

Good call on the sendRandomAction. This is why I wanted you to re-review.

Do we have a method that does a non-manual action?

Reply:
good question. We can `launch` some activity and then use [Espresso](https://developer.android.com/training/testing/espresso/basics) to make a tap on some element. And then if interactions tracking is disabled we shouldn't see it in RUM.

But maybe it is too much for such small method. I'm ok with removing E2E test as well. WDYT?